src/Octoshift/OrganizationMigrationStatus.cs

```diff
@@ -12,7 +12,7 @@ public static class OrganizationMigrationStatus
         public const string RepoMigration = "REPO_MIGRATION";
 
         public static bool IsSucceeded(string migrationState) => migrationState?.Trim().ToUpper() is Succeeded;
-        public static bool IsFailed(string migrationState) => migrationState?.Trim().ToUpper() is Failed;
-        public static bool IsPending(string migrationState) => !IsFailed(migrationState) && !IsSucceeded(migrationState);
+        public static bool IsPending(string migrationState) => migrationState?.Trim().ToUpper() is Queued or InProgress or NotStarted or PostRepoMigration or PreRepoMigration or RepoMigration;
+        public static bool IsFailed(string migrationState) => !(IsPending(migrationState) || IsSucceeded(migrationState));
```

Comment:
I think this should be && not ||

Reply:
I have switched it from `!A && !B` to `!( A || B )` to be consistent with the logic in `RepositoryMigrationStatus`.

Reply:
you're right, I missed that first !